diff --git a/dist/ui/operation/index.js b/dist/ui/operation/index.js
index 1e4d76a73695c022902e8c9314212856de677d11..2c17e99d0082b3af43c9d3734008e935425b2056 100644
--- a/dist/ui/operation/index.js
+++ b/dist/ui/operation/index.js
@@ -97,6 +97,7 @@ function Operation({ type = "operation", path, operation, pathItem, method, show
 						children: t("Request Body")
 					}), contentTypes.length > 1 ? /* @__PURE__ */ jsx(SelectTabTrigger, {
 						items,
+						"aria-label": "Request body content type",
 						className: "font-medium"
 					}) : /* @__PURE__ */ jsx("p", {
 						className: "text-fd-muted-foreground not-prose",
@@ -200,7 +201,10 @@ function Operation({ type = "operation", path, operation, pathItem, method, show
 					depth: headingLevel,
 					className: "my-0!",
 					children: t("Authorization")
-				}), items.length > 1 ? /* @__PURE__ */ jsx(SelectTabTrigger, { items }) : /* @__PURE__ */ jsx("div", {
+				}), items.length > 1 ? /* @__PURE__ */ jsx(SelectTabTrigger, {
+					items,
+					"aria-label": "Authorization scheme"
+				}) : /* @__PURE__ */ jsx("div", {
 					className: "not-prose",
 					children: items[0].label
 				})]
@@ -441,6 +445,7 @@ function ResponseAccordion({ status, operation }) {
 				children: items[0].label
 			}) : items.length > 0 && /* @__PURE__ */ jsx(SelectTabTrigger, {
 				items,
+				"aria-label": "Response content type",
 				className: "my-1.5 py-1"
 			})] }), /* @__PURE__ */ jsxs(AccordionContent, {
 				className: "ps-4.5 pe-3 border rounded-xl",
